content/version/2/1/code_of_conduct.id.md

```diff
@@ -0,0 +1,140 @@
+perwakilan komunitas seperti menggunakan alamat surel yang resmi, mengirim
+melalui akun media sosial resmi, atau sebagai perwakilan yang ditunjuk pada
+kegiatan daring maupun luring.
+
+## Penegakan
+
+Tindakan kasar, pelecehan, atau perilaku tidak dapat diterima lainnya yang
+terjadi dapat dilaporkan kepada pengelola komunitas di [MASUKKAN METODE KONTAK]
+untuk diselesaikan.
+Semua keluhan akan ditinjau dan diselidiki secara tepat dan adil.
+
+Semua pengelola komunitas berkewajiban untuk menjaga kerahasiaan dan keamanan
+pelapor suatu insiden.
+
+## Pedoman Penegakan
+
+Pengelola komunitas akan mengikuti Pedoman Dampak Komunitas berikut dalam
+menentukan konsekuensi untuk setiap tindakan yang dianggap melanggar dari Kode
+Etik ini:
+
+### 1. Koreksi
+
+**Dampak Komunitas**: Penggunaan bahasa yang tidak pantas atau perilaku lain
+yang dianggap tidak profesional atau tidak pantas dalam komunitas.
+
+**Konsekuensi**: Peringatan tertulis dari pengelola komunitas secara pribadi,
+meliputi penjelasan mengenai pelanggaran dan penjelasan mengapa perilaku itu
+tidak pantas. Permintaan maaf secara publik mungkin akan diminta.
+
+### 2. Peringatan
+
+**Dampak Komunitas**: Suatu pelanggaran melalui insiden tunggal atau serangkaian
+tindakan.
+
+**Konsekuensi**: Peringatan serta konsekuensi bagi yang masih melanggar. Tidak
+ada interaksi dengan orang-orang yang terlibat, termasuk interaksi yang tidak
+diminta dengan mereka yang menegakkan Kode Etik ini, dalam tempo waktu yang
+telah ditentukan. Hal ini juga termasuk menghindari interaksi di ruang komunitas
+serta saluran lainnya seperti media sosial. Melanggar ketentuan ini dapat
+menyebabkan pemblokiran sementara atau permanen.
+
+### 3. Pemblokiran Sementara
+
+**Dampak Komunitas**: Suatu pelanggaran serius terhadap standar komunitas,
+termasuk perilaku tidak pantas yang berkelanjutan.
+
+**Konsekuensi**: Pemblokiran sementara dari segala bentuk interaksi atau
+komunikasi publik dengan komunitas untuk jangka waktu tertentu. Tidak ada
+interaksi publik atau pribadi dengan orang-orang yang terlibat, termasuk
+interaksi yang tidak diminta dengan mereka yang menegakkan Kode Etik ini,
+diizinkan selama periode ini. Melanggar ketentuan ini dapat menyebabkan
+pemblokiran permanen.
+
+### 4. Pemblokiran Permanen
+
+**Dampak Komunitas**: Menunjukkan pola pelanggaran terhadap standar komunitas,
+termasuk perilaku tidak pantas yang berlanjut, pelecehan terhadap seorang
+individu, atau agresi atau penghinaan terhadap kasta dari individu tertentu.
+
+**Konsekuensi**: Pemblokiran permanen dari segala bentuk interaksi publik dalam
+komunitas.
+
+## Atribusi
+
+Kode Etik ini diadaptasi dari [Contributor Covenant][homepage], versi 2.1,
+tersedia di
+[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
+
+Pedoman Dampak Komunitas terinspirasi dari
+[Tahapan Pelaksanaan Kode Etik Mozilla][Mozilla CoC].
+
+Untuk jawaban atas pertanyaan umum tentang kode etik ini, lihat FAQ di
```

Comment:
```suggestion
Untuk jawaban atas pertanyaan umum tentang kode etik ini, lihat SSD di
```

Based on Language Development and Fostering Agency, FAQ in Indonesia would be [SSD, Soal Sering Ditanya](https://pasti.kemdikbud.go.id/istilah_listdet.php?id=109008&char=6&page=163). Although the equivalent word for it is indeed a formal/standard, it is rarely used by public.

Reply:
For me, stick with FAQ is better, at least for now. Using SSD term instead of FAQ will confuse the reader.